CHANGELOG.md

```diff
@@ -0,0 +1,215 @@
+# Changelog
+All notable changes to this project will be documented in this file.
```

Comment:
`s/will be/are/`?

Reply:
I'd say

This file documents all notable changes to the project since runc 1.0.

Reply:
I copied the header from https://keepachangelog.com/en/1.0.0/, but we can definitely customize it. I don't think it sounds wrong to use the future tense, since it's a living document, but either works (though the benefit of future tense is that it also acts as an instruction for contributors -- though that's not necessary).